### PR TITLE
Create test README.md

### PR DESCRIPTION
**Pull request by ArgusLi**
### Summary

Create test README.md

### Description

Created a README for tests.

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    DREMIO_USERNAME=
+    DREMIO_PASSWORD=
+    DREMIO_DATALAKE=dbt_test_source
+    DREMIO_DATABASE=dbt_test
+    DBT_TEST_USER_1=dbt_test_user_1
+    DBT_TEST_USER_2=dbt_test_user_2
+    DBT_TEST_USER_3=dbt_test_user_3
+    ```
+1. Create the three users listed above (dbt_test_user_1, dbt_test_user_2, dbt_test_user_3) in the Dremio instance.
+1. Create the source to be tested with and name it dbt_test_source.
```
Comment (jlarue26) [inline tests/README.md:21]:
```suggestion
1. Create an object storage source in Dremio called `dbt_test_source`.
```

Comment (jlarue26) [reply tests/README.md:21]: Perhaps link to this? https://docs.dremio.com/software/data-sources/gcs/

**Comment by jlarue26** [comment]
Add instructions to install dev-requirements.txt

File: tests/README.md (hunk 1 above)
Comment (ArgusLi) [reply tests/README.md]: Done and done.

**Review by jlarue26 — APPROVED** [review]
Looks good, thanks Argus.

**Pull request merged**